frontend/src/components/v1/ContractHealthRibbon.css

```
@@ -0,0 +1,135 @@
+  height: var(--chr-dot-size);
+  border-radius: 50%;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.6rem;
+  font-weight: 700;
+}
+
+/* Full variant: solid coloured dot */
+.chr--full .chr__dot {
+  background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)